app/models/finance/statement_line_item.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Finance::StatementLineItem < ApplicationRecord
+  self.table_name = "statement_line_items"
+
+  belongs_to :statement
+  belongs_to :participant_declaration
+
+  scope :eligible, -> { where(state: "eligible") }
+  scope :payable, -> { where(state: "payable") }
+  scope :paid, -> { where(state: "paid") }
+  scope :voided, -> { where(state: "voided") }
```

Comment:
can we have an `enum :state {}` to be consistent with the rest of the app and obvious what are the possible states?
That would also generate those scopes automatically.